content/courses/Ada_For_The_Embedded_C_Developer/chapters/06_Translation.rst

```diff
@@ -1548,8 +1548,29 @@ in the source object is automatically reflected in the target object (and
 vice-versa). In the end, the choice between unchecked conversions and overlays
 depends on the level of performance that you want to achieve.
 
-Also note that :ada:`Unchecked_Conversion` can only be instantiated for
-constrained types. In order to rewrite the examples using bit-fields that we've
+Also note that :ada:`Unchecked_Conversion` will only have defined behavior
```

Comment:
A side comment, not important in itself. Since the name of the generic is Ada.Unchecked_Conversion, and since Unchecked_Conversion is still a valid unit name, I suggest you say "unchecked conversion" when you're talking about the semantic effect rather than the unit name.

Reply:
Yes, I agree. I've just changed a few instances of it in the text.